### ISSUE TITLE
[SF][A11Y][Badge]: Fails to meet color contrast ratio with respect to background color when color scheme is "1"

### ISSUE THREAD

**Issue by sfsf-xweb-sh** [SAP/ui5-webcomponents#3940]
Hi Colleague,
There is a A11Y issue:Fails to meet color contrast ratio with respect to background color when color scheme is "8". 
Please take a look. This issue affect OMP and Assignment(b2111 release).

![Screen Shot 2021-09-16 at 5 40 08 PM](https://user-images.githubusercontent.com/78520316/133591040-2a355d95-cf5c-4d0e-8909-07d0ec63dc23.png)

### **Expected Behavior**
![Screen Shot 2021-09-16 at 5 44 31 PM](https://user-images.githubusercontent.com/78520316/133590915-af4aee4a-ff12-44dd-81a9-1ee71bb02dd3.png)

### **Context**
- UI5 Web Components version: {0.31.17}
- Browser: {chrome}

### **Priority**
- [ ] High

**Comment by ilhan007** [comment]
Hello @SAP/ui5-webcomponents-topic-rd could you check the contrast ration issue that has been reported. Perhaps this might be fixed with the latest color changes.

https://github.com/SAP/ui5-webcomponents/pull/3926

**Comment by ilhan007** [comment]
Hello @sfsf-xweb-sh the title mentions scheme "1", the issue description - scheme "8". Is it 1 or 8 at the end.

**Comment by gmkv** [comment]
The one pictured is `color-scheme="1"`. And it indeed fails to meet the minimum contrast ratio. I'll discuss this with the design team.

**Comment by gmkv** [comment]
Hi @sfsf-xweb-sh 
The design team advised that when the text inside the Badge is bold, it is acceptable to have lower contrast ratio.
This is described here https://wiki.wdf.sap.corp/wiki/pages/viewpage.action?pageId=2095479022#Accessibility(Fiori3)-ACC-261-MinimumContrast

Regards,
Georgi

**Comment by sfsf-xweb-sh** [comment]
> Hello @sfsf-xweb-sh the title mentions scheme "1", the issue description - scheme "8". Is it 1 or 8 at the end.

Sorry for typo error, It's 1 in this issue.

**Comment by sfsf-xweb-sh** [comment]
Hi @gmkv @ilhan007 , Could you double check this with @Anu Sandhanam(I524170)

**Comment by gmkv** [comment]
Hi @sfsf-xweb-sh 

The component's colors were updated. The fix will be available with the next release of ui5-webcomponents.
